src/Compilers/Test/Utilities/CSharp/SymbolUtilities.cs

```diff
@@ -93,9 +93,14 @@ public static string ToTestDisplayString(this ISymbol symbol)
             return CodeAnalysis.Test.Extensions.SymbolExtensions.ToTestDisplayString(symbol);
         }
 
-        public static string ToTestDisplayString(this SymbolWithAnnotations symbol)
+        public static string ToTestDisplayString(this SymbolWithAnnotations symbol, bool includeNonNullable = false)
```

Comment:
>ToTestDisplayString [](start = 29, length = 19)

Should this new extension method be moved to the "right" place (see comment on method above)?

Reply:
It looks like there are more callers of the extension method above than the extension method referenced in the comment. Perhaps the extension method in the comment should be replaced instead.

---
In reply to: [197936968](https://github.com/dotnet/roslyn/pull/27525#discussion_r197936968) [](ancestors = 197936968)